R/zchunk_LA114.wind-energy.R

```diff
@@ -72,35 +70,16 @@ module_energy_LA114.wind <- function(command, ...) {
       (L114.supply_tol / (1-L114.supply_tol)) ^ (1 / curve.exponent) * mid.price + base.price # max price
     }
 
-    if(OLD_DATA_SYSTEM_BEHAVIOR) {
-
-      # Romania is listed in the Zhou file as rou and in the iso_GCAM_regID file as rom. Consequently, it was dropped in the old code.
-      # We are keeping Romania in the new code.
-
-      # Country-level supply curves from the Zhou file are used to build up the regional supply curves
-      Zhou_wind_supply_ctry_EJ %>%
-        rename(mid.price = `mid-price`, curve.exponent = `curve-exponent`, base.price = `base-price`) %>% # Modifying title names
-        # Romania will be purposefully dropped using inner_join when attaching region IDs to the countries
-        inner_join(iso_GCAM_regID, by = "iso") %>%
-        # Convert prices per unit of energy from 2007 USD/kWh to 1975 USD/GJ
-        mutate(mid.price = mid.price * gdp_deflator(1975, 2007) / CONV_KWH_GJ,
-               base.price = base.price * gdp_deflator(1975, 2007) / CONV_KWH_GJ) ->
-        L114.RsrcCurves_EJ_ctry_wind
-
-    } else {
-
-      # Country-level supply curves from the Zhou file are used to build up the regional supply curves
-      Zhou_wind_supply_ctry_EJ %>%
-        rename(mid.price = `mid-price`, curve.exponent = `curve-exponent`, base.price = `base-price`) %>% # Modifying title names
-        # Romania is listed in the Zhou file as rou and in the iso_GCAM_regID file as rom. rou will be changed to rom.
-        mutate(iso = replace(iso, iso == "rou", "rom")) %>%
-        left_join_error_no_match(iso_GCAM_regID, by = "iso") %>% # Attach region IDs to the countries
-        # Convert prices per unit of energy from 2007 USD/kWh to 1975 USD/GJ
-        mutate(mid.price = mid.price * gdp_deflator(1975, 2007) / CONV_KWH_GJ,
-               base.price = base.price * gdp_deflator(1975, 2007) / CONV_KWH_GJ) ->
-        L114.RsrcCurves_EJ_ctry_wind
-
-    } # End of old system behavior function
+    # Country-level supply curves from the Zhou file are used to build up the regional supply curves
+    Zhou_wind_supply_ctry_EJ %>%
+      rename(mid.price = `mid-price`, curve.exponent = `curve-exponent`, base.price = `base-price`) %>% # Modifying title names
+      # Romania is listed in the Zhou file as rou and in the iso_GCAM_regID file as rom. rou will be changed to rom.
+      mutate(iso = replace(iso, iso == "rou", "rom")) %>%
```

Comment:
Do we want to open an issue to fix this?

Reply:
Ehh, so easy and doesn't affect anything else:
```
WE34383:server2 pralitp$ diff -q outputs/L114.RsrcCurves_EJ_R_wind.csv compdata_updates/L114.RsrcCurves_EJ_R_wind.csv 
WE34383:server2 pralitp$
```

Might as well take care of it now.